### PR TITLE
NAS-110130 / 21.06 / Fix mountpoint issue on storage page.

### PR DESCRIPTION
**Pull request by dszidi**
To test:
- Make sure that pool.dataset.query is not returning mountpoints as null
- make sure all "Edit Permissions" links work and take the user to the correct form (acl/posix/permissions)

### CODE REVIEW

File: src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts [hunk 1]
```diff
@@ -1100,13 +1100,14 @@ export class VolumesListTableConfig implements InputTableConf {
...
           onClick: () => {
-            this.ws.call('filesystem.acl_is_trivial', ['/mnt/' + rowData.id]).subscribe((acl_is_trivial) => {
+            this.ws.call('filesystem.acl_is_trivial', [rowData.mountpoint]).subscribe((acl_is_trivial) => {
               if (acl_is_trivial) {
                 this._router.navigate(new Array('/').concat([
                   'storage', 'permissions', rowData.id,
                 ]));
               } else {
                 this.ws.call('filesystem.getacl', [rowData.mountpoint]).subscribe((acl) => {
+                  if (rowData.id.includes('test_homes')) console.log(acl);
```
Comment (william-gr) [inline src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts:1110]: leftover debug?

Comment (dszidi) [reply src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts:1110]: Oops. Yep will remove.

**Comment by undsoft** [comment]
Review blocked by https://github.com/truenas/middleware/pull/6927

**Comment by william-gr** [comment]
We can go ahead with this, related PR merged.

**Comment by undsoft** [comment]
I'll retest it tomorrow, once the updated nightly instance is there.

**Comment by william-gr** [comment]
@undsoft we are going into freeze today, you can get an incremental build here https://ci.tn.ixsystems.net/jenkins/job/TrueNAS%20SCALE%20-%20Unstable/job/Build%20-%20TrueNAS%20SCALE%20(Incremental)/

**Review by undsoft — APPROVED** [review]
Tested on incremental build and fixed another random issue.

**Pull request merged**